bodhi/server/captcha.py

```diff
@@ -1,281 +0,0 @@
-# Copyright © 2014-2018 Red Hat, Inc.
-#
-# This file is part of Bodhi.
-#
-# This program is free software; you can redistribute it and/or
-# modify it under the terms of the GNU General Public License
-# as published by the Free Software Foundation; either version 2
-# of the License, or (at your option) any later version.
-#
-# This program is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with this program; if not, write to the Free Software
-# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
-# 02110-1301, USA.
-#
-# Authors:  Ralph Bean <rbean@redhat.com>
-"""Define utilities and a view pertaining to captcha images for unauthenticated users."""
-
-import base64
-import math
-import random
-import binascii
-
-from PIL import Image, ImageDraw, ImageFont
-from pyramid.httpexceptions import HTTPGone, HTTPNotFound
-from pyramid.view import view_config
-import cryptography.fernet
```

Comment:
We should remove PIL and cryptography from the dependencies in ```requirements.txt```, and we should also add a note to the release notes about the dropped dependencies.